core/conn/odb/src/odb.c

```diff
@@ -13133,7 +13134,7 @@ static int Otcol(int eid, SQLHDBC *Ocn)
                             str[j]= '\0';
                         }
                     } else {
-                        if ( isspace((int)str[j]) || str[j]=='#' )
```

Comment:
Thanks for your explanation.
With this change odb will not support comment in the pamfile, so [TRAFODION-2102] is not a bug is not a bug, and I will close that jira.